sdk/core/azure-core/src/http/retry_policy_private.hpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "azure/core/http/http.hpp"
+#include "azure/core/http/policies/policy.hpp"
+
+#include <chrono>
+
+namespace Azure { namespace Core { namespace Http { namespace Policies { namespace _detail {
+
+  bool ShouldRetryOnTransportFailure(
+      RetryOptions const& retryOptions,
+      int32_t attempt,
+      std::chrono::milliseconds& retryAfter,
+      double jitterFactor = -1);
+
+  bool ShouldRetryOnResponse(
+      RawResponse const& response,
+      RetryOptions const& retryOptions,
+      int32_t attempt,
```

Comment:
const?

Reply:
If this was a non-static class member - absolutely, but today it is a free function, and according to your feedback I am going to move it to a "static class".